style.css

```diff
@@ -126,6 +126,71 @@ header {
   align-items: center;
 }
 
+.modal-pop {
+  position: fixed;
+  top: -25%;
+  left: 45%;
+  transform: translate(-50%, 50%) scale(1);
+  border-radius: 10px;
+
+  /* opacity: 0; */
+  z-index: 10;
+  width: 500px;
+  max-width: 80%;
+}
+
+.card-item-modal {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  width: 327px;
+  padding: 16px;
+  border-radius: 16px;
+  background-color: var(--main-bg-color);
+}
+
+.modal-pop.active {
+  transform: translate(-50%, 50%) scale(1);
+  top: -25%;
+  left: 45%;
+}
+
+.modal {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 2rem;
+  padding: 5px 5px;
+}
+
+.close-x {
+  font-size: 2.25rem;
+  font-weight: bold;
+  background: none;
+  background-color: #fff;
+  cursor: pointer;
+  outline: none;
+  border: none;
+  margin-left: 10rem;
+}
+
+#overlay {
+  position: fixed;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
- [ ] While you made efforts, your design is presently not in place, and also there's a static pop-up on both the mobile and desktop. Kindly check out the given design from this [TEMPLATE](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) and fix the adjustments, as the main design needs to be put together 👍 

# Your current view mobile  :
![image](https://user-images.githubusercontent.com/50990925/216019893-10919a89-4273-419f-93f2-145e3d70c720.png)

# Expected figma design mobile:
![image](https://user-images.githubusercontent.com/50990925/216019977-35dcb969-ab98-4362-a12e-442f468fb832.png)

# Current view Desktop with a static and unresponsive pop-up:
![image](https://user-images.githubusercontent.com/50990925/216020269-f7080509-92b6-4be2-bcb7-6da43cf660f9.png)

# Expeeted design with a responsive close button for a pop-up to close:
![image](https://user-images.githubusercontent.com/50990925/216020528-3b69177c-5658-4862-a90c-d99825ac128f.png)